core/src/main/scala/org/apache/spark/rapids/tool/benchmarks/README.md

```diff
@@ -0,0 +1,32 @@
+# Benchmarking Tools
+
+This package contains the relevant classes to write and run benchmarks for `RAPIDS TOOLS` for Apache Spark.
+
+## Writing a benchmark -
+* Extend `BenchmarkBase` and override the `runBenchmarkSuite` function
+* Write logically similar benchmarks inside the `runBenchmark` which will add a header to the output with the name provided.
+* Now for each case to be tested, use the Benchmark class `addCase` to write various cases of the same Benchmark.
+* Call the run function to run the benchmark
+* Refer included example benchmark - `SingleThreadedQualToolBenchmark` for implementation details
```

Comment:
It might be a good idea to create a link to the example file

Reply:
Updated